task/blue-green-deploy/0.1/README.md

```diff
@@ -0,0 +1,86 @@
+# Blue Green Deploy on Kubernetes
+
+The following task can help you to deploy an application using the Blue-Green deployment strategy. This task gives you the flexibility to deploy a version of the application from the blue zone to the green zone without disturbing the previous version in the blue zone. It works by making the service point to the newer version of the application deployed.
+
+- If you are deploying the first version of the application then it will go into the blue zone and for that you need to provide deployment and service manifests via the `workspaces` and in the `params` the service name and version. Sample Kubernetes manifests for version v1 can be found [here](./example/v1-deploy).
+- For further new deployments, it will get deployed in the other zone and then the current service will now point to the new deployment. For example if we have a deployment running in the blue zone then we will deploy the next deployment in the green zone and make the service point to the green zone or vice versa.
+
+## Installing the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/blue-green-deploy/0.1/blue-green-deploy.yaml
+```
+
+## Installing the ClusterRoleBinding
```

Comment:
I think you need to keep this file under different directory from `task` as the kind is different. So your `clusterrolebinding.yaml` will be in a new directory following a structure as `/ClusterRoleBinding/default-blue-green/0.1/default-blue-green.yaml`

cc @sthaha  @vdemeester